spec/models/page_answers_spec.rb

```diff
@@ -120,6 +120,109 @@
         end
       end
     end
+
+    context 'when sanitizing answers' do
+      context 'when the answer should not be sanitised' do
+        context 'when component type is text' do
+          let(:answers) { { 'name_text_1' => 'script/' } }
+
+          it 'returns the value of the answer' do
+            expect(page_answers.name_text_1).to eq('script/')
+          end
+        end
+
+        context 'when the component is a textarea' do
+          let(:page) { service.find_page_by_url('family-hobbies') }
+          let(:answers) { { 'hobbies_textarea_1' => 'script/' } }
```

Comment:
can we make this part of a paragraph please. it's probably all fine but just wanna make sure it's not gonna rip out a random script word